source/cppexpose/include/cppexpose/reflection/Object.h

```diff
@@ -55,6 +55,9 @@ class CPPEXPOSE_API Object : public AbstractProperty
     */
     Object(const std::string & name);
 
+    Object(const Object &) = delete;
```

Comment:
Not documenting is also an anti-pattern, but somehow nobody cares about that. But beware of documentation anti-patterns ...